tensorflow/core/platform/default/logging.cc

```diff
@@ -31,28 +31,41 @@ limitations under the License.
 #include <string.h>
 #include <time.h>
 
-#include <string>
+#include <algorithm>
+#include <mutex>
```

Comment:
Please use tensorflow/core/platform/mutex.h instead of <mutex> directly. Then replace std::mutex with (tensorflow::)mutex and std::lock_guard with (tensorflow::)mutex_lock.